raw_data_gen.py

```diff
@@ -0,0 +1,140 @@
+DB_DIRS = ["./data_hf", "./data_tb", "./data_timm"]
+
```

Comment:
We should eventually move most of the feature generation in this file to happen at codegen time.  Though something like this could be useful for experimenting on what features are useful.